pkg/inventory/inventory_test.go

```diff
@@ -423,7 +423,7 @@ func copyInventory() InventoryInfo {
 
 func storeObjsInInventory(info InventoryInfo, objs object.ObjMetadataSet) *unstructured.Unstructured {
```

Comment:
This function is only used in `inventory-client_test.go`. I think you can move it into that file. Then it probably needs to be modified to take the status too, and those tests modified to also store the status.

Reply:
done